playbooks/generic-build_images.yml

```diff
@@ -1,8 +1,12 @@
 ---
-- hosts: osh-deployer
+- hosts: osh-deployer[0]
```

Comment:
There could be more deployers?

Reply:
I guess it will depend on how we do things with airship, either it's the same node or not, either we name those the same way or not.

But the idea behind this is to ensure this rolls on a single node at all time, and that would be the node for which we grab the facts. Not another one.

Reply:
I understand we can theoretically have more deployer nodes, I'm just suprised that only this playbook specifically chooses the first item

Reply:
That's because that's the only one to rely on a fact that's set on the first node :)